lib/pf/services/manager/pfacct.pm

```diff
@@ -58,15 +59,21 @@ Generate the environment variables for running the container
 
 sub generate_container_environments {
     my ($self, $tt) = @_;
-    my $management_ip = $management_network->tag('ip');
 
     my $port = '1813';
-    if ($cluster_enabled) {
+    my $listeningIp = "";
+    if ($cluster_enabled || isenabled($Config{services}{radiusd_acct})) {
         $port = '1823';
     }
+    if ($cluster_enabled && isenabled($Config{services}{radiusd_acct})) {
+        $port = '1833';
+    }
+    if (isenabled($Config{services}{radiusd_acct})) {
+        $listeningIp = '127.0.0.1';
+    }
     my $vars = {
        env_dict => {
-           PFACCT_ADDRESS=> "$port",
+           PFACCT_ADDRESS=> "$listeningIp:$port",
```

Comment:
Is the "$port" = ":$port" when $listeningIp = ""?
For me, PFACCT_ADDRESS=:1833 is now different then the previous results

Reply:
the " : " should be added to $listeningIp when not empty